ang/crmCaseType.css

```diff
@@ -32,6 +34,14 @@
     width: 10em;
 }
 
+.crmCaseType-edit-actset {
```

Comment:
Please use `.crmCaseType__activitySet-label--edit`:

* no need to shorten the name.
* you could either use BEM `.crmCaseType__activitySet-label--edit` or follow the convention in the file: `.crmCaseType .edit-activitySet-label`.

Reply:
Following existing convention, `crmCaseType-edit-activity-set`